Cargo.toml

```diff
@@ -17,6 +17,8 @@ linux-native = ["dep:linux-keyutils"]
 apple-native = ["dep:security-framework"]
 windows-native = ["dep:windows-sys", "dep:byteorder"]
 
+sync-persistent-keyutils = ["dep:linux-keyutils", "sync-secret-service"]
```

Comment:
should be:
```
sync-persistent-keyutils = ["linux-native", "sync-secret-service"]
```
and same change for async.  This is what I meant and what makes the original config work for loading keyutils and choosing the default.

It makes even more sense if you rename this feature to "linux-native-sync-persistent" and the next one to "linux-native-async-persistent".